worker/bin/task-script-executor.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# Read the task script file and execute one command at a time, saving your exitcodes in the .ts.ec file.
+# Each command executed is written to the .cmds file.
+# Use -tsf= or --task_filepath= to input the task file path (Required).
+# Use the flag -d or --debug to store .out and .err from execution (Optional).
+
+# This flag does the execution not stop on non-zero exit code commands
+set +e
+
+EXECUTION_DIR=/arrebol
```

Comment:
could we use a more significant name for the EXECUTION_DIR?

Reply:
maybe work_dir?